cmd/rpcdaemon/commands/eth_accounts.go

```diff
@@ -119,3 +119,27 @@ func (api *APIImpl) GetStorageAt(ctx context.Context, address common.Address, in
 	}
 	return hexutil.Encode(common.LeftPadBytes(res, 32)), err
 }
+
+// Exist returns whether an account for a given address exists in the database.
+func (api *APIImpl) Exist(ctx context.Context, address common.Address, blockNrOrHash rpc.BlockNumberOrHash) (*hexutil.Uint, error) {
```

Comment:
1. Can it return bool instead `hexutil.Uint` ?
2. does `eth_GetTransactionCount` return 0 for non-existing accounts? Or it can return 0 for non-existing also? I mean maybe client can do something like: `exists := eth_GetTransactionCount() == 0`?

Reply:
1. Yeah
2. Unfortunately no, someone sending 0 ether to the address will invoke `CreateAccount()` function for that account and persist it in the database. From that point on, `eth_exist` will return `true` and `eth_getTransactionCount` will still return `0`.

Reply:
@nebojsa94 do you mind if we will move it from `eth_exist` to `erigon_exist`?

Reply:
@AskAlexSharov This is the core component of Ethereum's state, therefore I think it should be a part of `eth` namespace. Also, I think other clients should implement this as well.

Is there any specific reason for the namespace change?

Reply:
Just to see where we compatible with Geth - and where is something our own. 
But I think it's fine. Let's merge.